src/components/organize/journeys/JourneyCard.tsx

```diff
@@ -18,31 +11,36 @@ interface JourneyCardProps {
 
 const JourneyCard = ({ journey }: JourneyCardProps): JSX.Element => {
   const { orgId } = useRouter().query;
-  const { id, singular_label, stats } = journey;
+  const { id, title, stats } = journey;
 
```

Comment:
was there not a discussion somewhere about using singular_label insted of title? or was it the other way around?

Reply:
It was that we should use title here